### PR TITLE
added polish translation

### PR DESCRIPTION
**Pull request by paulinek13**

### CODE REVIEW

File: dist/_locales/pl/messages.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    "description": ""
+  },
+  "hour": {
+    "message": "h",
+    "description": ""
+  },
+  "hours": {
+    "message": "h",
+    "description": ""
+  },
```
Comment (okaryo) [inline dist/_locales/pl/messages.json:49]: Which one is more commonly used in Polish, 'h' or 'godziny'?
For example, in English, it is used in UI texts like '15 hours ago' to indicate elapsed time.

```json
  "hour": {
    "message": "godzina",
    "description": ""
  },
  "hours": {
    "message": "godziny",
    "description": ""
  },
```

**Review by okaryo — COMMENTED** [review]
Thank you for Pull Request!

I have one question!

File: dist/_locales/pl/messages.json (hunk 1 above)
Comment (paulinek13) [reply dist/_locales/pl/messages.json:49]: Yes, "godziny" is more commonly used, but it's **a little bit** more complicated too (I assume those messages try to give an answer to _when?_):

- _1 godzinę temu_
- _2 godziny temu_
- _8 godzin temu_
- _24 godziny temu_
- _36 godzin temu_
- ...

I mean - there are rules for that, but I wanted to keep it simple.
_Now_, I think **the best** solution is to use **_godz._** for both singular and plural.

Comment (okaryo) [reply dist/_locales/pl/messages.json:49]: Thank you for letting me know!
Now let's go with `godz`!

**Review by okaryo — APPROVED** [review]
LGTM!

I will release the latest version of TabTabTab in a few days, so you can use TabTabTab in Polish if you like. If you have any suggestions for improvements to the functionality, please feel free to contribute!

**Comment by okaryo** [comment]
#48

**Pull request merged**